gradio/components/image.py

```diff
@@ -83,7 +106,7 @@ def __init__(
         """
         Parameters:
             value: A PIL Image, numpy array, path or URL for the default value that Image component is going to take. If callable, the function will be called whenever the app loads to set the initial value of the component.
-            format: File format (e.g. "png" or "gif") to save image if it does not already have a valid format (e.g. if the image is being returned to the frontend as a numpy array or PIL Image). The format should be supported by the PIL library. Applies both when this component is used as an input or output. This parameter has no effect on SVG files.
+            format: File format (e.g. "png" or "gif") or "base64". Used to save image if it does not already have a valid format (e.g. if the image is being returned to the frontend as a numpy array or PIL Image). If not "base64", the format should be supported by the PIL library. Applies both when this component is used as an input or output. This parameter has no effect on SVG files.
```

Comment:
The issue I see with using `format` is that you might want to set a particular file extension _and_ transmit it using base64, no?

Reply:
It also doesn't really make sense given the description of this parameter:  "Used to save image if it does not already have a valid format"

Reply:
In fact, here's a totally different idea, what if we actually add a `use_base64: bool = True` parameter to the `.stream()` event and use it for Audio/Image input streaming?

Reply:
That way, most users don't even have to think about it? It just gets used under the hood